kylexrich/coursegpt
# 19 school course controller

Addresses #19 
### What this does:
- adds school/course selection logic for users, appearing as a settings popup

### How to test:
- the popup should automatically come up when you are redirected after login. An alternative is to click the hamburger icon next to the username and click on the profile button
- after selecting a school, the courses associated with that school should pop up. After choosing courses, they should be available as normal courses on the top left; and you can switch between then (with the respective prompts switching too - the prompts were written half sleep-deprived so some of them are not the greatest)

### Who this affects:
- Definitely should've given a heads up earlier, but this affects anyone working on the user/settings page (I think that's Ritik), as well as anyone working on main page components that involve courses, the currently selected course, or schools.
